<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagManager</class>
 <widget class="QWidget" name="TagManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #484848;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #cccccc;</string>
     </property>
     <property name="text">
      <string>TAG MANAGER</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="result_label">
       <property name="styleSheet">
        <string notr="true">color: #e2c08d;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ClickableQLabel" name="icon_mode_label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/Resources/icon/tag-5-16.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableQLabel" name="list_mode_label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/Resources/icon/list-2-16.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableQLabel" name="update_label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/Resources/icon/reload.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableQLabel" name="add_label">
       <property name="statusTip">
        <string>Add Tag</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/Resources/icon/plus-2-16.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="filter_line_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #cccccc; background-color: #383838;</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="list_view">
     <property name="styleSheet">
      <string notr="true">color: #cccccc; background-color: #383838;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableQLabel</class>
   <extends>QLabel</extends>
   <header>libs/clickableQLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
